server/db/db.js

```diff
@@ -67,6 +68,11 @@ async function beginTransaction() {
 
     await pgClient.query('LOCK TABLE clients, sessions, locations')
   } catch (e) {
+    if (!isRetry) {
+      isRetry = true
```

Comment:
Hmmm, does this work? I would have expected `isRetry` to be `false` again each time you call `beginTransaction`.

Reply:
Hmm good point, I think instead I'll add a default parameter for the retryCount then use number for my conditional. Stay tuned!